ui/apps/platform/cypress/integration/systemHealth/declarativeConfiguration.test.js

```diff
@@ -0,0 +1,35 @@
+import withAuth from '../../helpers/basicAuth';
+import { visitSystemHealthWithStaticResponseForCapabilities } from '../../helpers/systemHealth';
+
+describe('Declarative Configuration', () => {
```

Comment:
You might expand as `'System Health Declarative Configuration'`

It helps to scan in case of integration test failures. For example, it was obvious that I made a serious mistake recently when titles were from 2 related test files:
* Compliance Dashboard and so on
* Compliance entities list and so on